themes/default/views/clients/home.blade.php

```diff
@@ -37,39 +37,26 @@ function removeElement(element) {
+                                            <span class="font-semibold">{{ $invoice->id }}</span>
+                                    @php
                                     $products = $invoice->getItemsWithProducts()->products;
                                 @endphp
-                                @foreach ($products as $item)
-                                    @php
-                                        $expiryDate = new \DateTime($item->expiry_date);
-                                        $countdownDate = clone $expiryDate;
-                                        $countdownDate->modify('+7 days')->setTime(0, 1, 0);
-                                        $currentDate = new \DateTime();
-                                        $interval = $currentDate->diff($countdownDate);
-                                        $remainingDays = $interval->days;
-                                        $remainingHours = $interval->h;
-                                    @endphp
-                                    <div class="hover:bg-secondary-300 transition-all border-b-[1px] border-secondary-200 ease-out hover:ease-in cursor-pointer w-full p-3" onclick="window.location.href='{{ route('clients.invoice.show', $invoice->id) }}'">
-                                        <div class="flex flex-row">
-                                            <div class="flex my-auto text-2xl ms-2 w-1/12 justify-center text-blue-500">
-                                                <i class="ri-bill-fill"></i>
-                                            </div>
-                                            <div class="w-8/12 ml-4 truncate">
-                                                <span class="font-semibold">{{ $item->product->name ?? $item->name }}</span>
-                                                <span class="font-semibold">(#{{ $item->product->id?? $item->id }})</span>
-                                                <span class="text-blue-500 items-center flex opacity-80 text-sm">{{__('Pay for the product / service')}}</span>
-                                            </div>
-                                            <div class="justify-end min-h-[48px] max-h-[48px] flex w-3/12 text-sm font-medium text-center text-red-500">
-                                                @if ($currentDate < $countdownDate)
-                                                    {{__('Time to pay')}} {{ $remainingDays }}d {{ $remainingHours }}h
-                                                @else
-                                                    {{ __('Expired')}}
-                                                @endif
-                                            </div>
+                                    <div class="w-full text-sm text-gray-400 truncate">
+                                        <span class="font-semibold">Amount to pay - {{ $invoice->total() }} {{ config('settings::currency_sign') }}</span>
```

Comment:
Use a language variable instead of hard coding it

Reply:
Corrected, thanks